src/NoFrixion.MoneyMoov/Models/Transaction/Transaction.cs

```diff
@@ -30,6 +30,11 @@ public class Transaction : IWebhookPayload
     /// </summary>
     public Guid AccountID { get; set; }
 
+    /// <summary>
+    /// The name of the account the transaction belongs to.
+    /// </summary>
+    public string AccountName { get; set; }

```

Comment:
Shouldn't this be nullable

Reply:
This file has `#nullable disable`, so it's not needed.